src/blockchain/xu3lp/prices.ts

```diff
@@ -34,7 +34,7 @@ import { getLPTokenSymbol } from '../utils'
  */
 export const getXU3LPPrices = async (
   xu3lpContract: XU3LP,
-  kyberProxyContract: KyberProxy
+  provider: BaseProvider
```

Comment:
You can access the `provider` from `xu3lpContract.provider` which makes this argument redundant. Please remove this and refactor this change across. Refer to other xAssets price methods for usage.